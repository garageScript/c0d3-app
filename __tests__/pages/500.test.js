import React from 'react'
import { render, waitFor } from '@testing-library/react'
import { MockedProvider } from '@apollo/client/testing'
import InternalServerError from '../../pages/500'

describe('500 Error Page', () => {
  it('should render', async () => {
    const { container } = render(
      <MockedProvider>
        <InternalServerError />
      </MockedProvider>
    )
    await waitFor(() => expect(container).toMatchSnapshot())
  })
})
