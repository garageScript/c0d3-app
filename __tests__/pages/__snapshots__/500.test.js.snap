// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`500 Error Page should render 1`] = `
<div>
  <nav
    class="navbar navbar navbar-expand-lg navbar-light bg-white"
    id="appNav"
  >
    <div
      class="container"
    >
      <a
        class="navbar-brand"
        href="/"
      >
        <div
          class="navbar-brand text-primary fw-bold"
        >
          C0D3
          
        </div>
      </a>
      <button
        aria-controls="basic-navbar-nav"
        aria-label="Toggle navigation"
        class="navbar-toggler collapsed"
        type="button"
      >
        <span
          class="navbar-toggler-icon"
        />
      </button>
      <div
        class="text-center navbar-collapse collapse"
        id="basic-navbar-nav"
      >
        <div
          class="m-auto navbar-nav"
        >
          <a
            class="nav-item nav-link px-3 noUnderline"
            href="/curriculum"
          >
            Curriculum
          </a>
          <a
            class="nav-item nav-link px-3 noUnderline"
            href="https://github.com/garageScript/c0d3-app"
            rel="noopener noreferrer"
            target="_blank"
          >
            Repo
          </a>
          <a
            class="nav-item nav-link px-3 noUnderline"
            href="/docs/learn"
          >
            Learn
          </a>
          <a
            class="nav-item nav-link px-3 noUnderline"
            href="https://discord.gg/c0d3"
            rel="noopener noreferrer"
            target="_blank"
          >
            Community
          </a>
        </div>
      </div>
    </div>
  </nav>
  <div
    class="container-md container"
  >
    <div
      class="container"
    >
      <img
        height="360"
        src="/assets/errors/500.svg"
        width="360"
      />
      <div
        class="container_text_buttons"
      >
        <h1>
          500 Error!
        </h1>
        <div
          style="color: rgb(41, 41, 41); font-size: 1.25rem;"
        >
          Internal server error
        </div>
        <div
          class="container_buttons"
        >
          <a
            class="button btn btn-primary py-2 px-3 mt-3 fw-bold noUnderline"
            href="/curriculum"
          >
            Home
          </a>
          <a
            class="button btn btn-primary py-2 px-3 mt-3 fw-bold noUnderline"
            href="/docs/learn"
          >
            Learn
          </a>
        </div>
      </div>
    </div>
  </div>
  <footer
    class="mt-5 fw-light text-center pb-5 text-muted"
  >
    Â© Copyright 
    2024
     GarageScript
  </footer>
</div>
`;
